Description: fix detection of Java versions >= 10
 When this version check was written, it was believed Java would never get
 beyond 9.  We're at 11 now, baby!
Author: Steve Langasek <steve.langasek@ubuntu.com>
Last-Modified: 2018-04-24

Index: db5.3-5.3.28/dist/configure.ac
===================================================================
--- db5.3-5.3.28.orig/dist/configure.ac
+++ db5.3-5.3.28/dist/configure.ac
@@ -454,7 +454,7 @@
 	esac
 	AC_MSG_RESULT($JAVA_VERSION)
 	case "$JAVA_VERSION" in
-	1.[[3456789]]* | 1.[[1-9]][[0-9]]* | [[23456789]]* ) ;;
+	1.[[3456789]]* | 1.[[1-9]][[0-9]]* | [[23456789]]* | [[123456789]][[0123456789]]*) ;;
 	* )
 		AC_MSG_ERROR([Java version 1.3 or higher required, got $JAVA_VERSION]) ;;
 	esac
